core/src/test/java/org/apache/iceberg/TestMetadataUpdateParser.java

```diff
@@ -325,7 +325,7 @@ public void testAddSnapshotToJson() {
 
     Snapshot snapshot = new BaseSnapshot(null, snapshotId, parentId, System.currentTimeMillis(),
         DataOperations.REPLACE, ImmutableMap.of("files-added", "4", "files-deleted", "100"),
-        schemaId, manifests);
+        schemaId, manifests, null);
```

Comment:
Rather than making so many changes to add `null` to tests, can you add a package-private constructor that defaults this?

Reply:
sure, adding